src/bindings/python/wheel/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(PUGIXML_LIBS_DIR runtime/3rdparty/pugixml/lib)
 # Dependencies
 #
 
-foreach(_target ie_api constants _pyngraph openvino_c pyopenvino ov_plugins ov_frontends py_ov_frontends)
+foreach(_target ie_api constants _pyngraph openvino_c pyopenvino ov_plugins ov_frontends py_ov_frontends inference_engine_snippets)
```

Comment:
`inference_engine_snippets` is a dependency of Intel CPU plugin.
If CPU plugin is not built as in your configuration, we should not build snippets as well.

I suppose we need to add `if(ENABLE_INTEL_CPU)` here https://github.com/openvinotoolkit/openvino/blob/master/src/common/CMakeLists.txt#L9

Reply:
Thank you so much for the review and suggestions!

I didn't quite understand one thing: you advise to add a condition for building snippets, right?  
But, if they are not built, - wouldn't the wheel build crush in that case?  
Or, do you mean this condition will remove an expected dependency from the wheel as well, so there will be no error?

Anyway, I'll experiment with that locally and come back with the results

Reply:
> Or, do you mean this condition will remove an expected dependency from the wheel as well, so there will be no error?

Yes

Reply:
@ilya-lavrenov thank you so much for the advise!
Checked locally, your way works perfectly.